tests/fork/merkle-campaign/MerkleVCA.t.sol

```diff
@@ -71,13 +71,13 @@ abstract contract MerkleVCA_Fork_Test is MerkleBase_Fork_Test {
             params: constructorParams,
             aggregateAmount: vars.aggregateAmount,
             recipientCount: vars.leavesData.length,
-            fee: vars.minimumFee,
+            minFeeUSD: vars.minFeeUSD,
             oracle: vars.oracle
         });
 
         merkleVCA = merkleFactoryVCA.createMerkleVCA(constructorParams, vars.aggregateAmount, vars.leavesData.length);
 
-        assertGt(address(merkleVCA).code.length, 0, "MerkleVCA contract not created");
+        assertLt(0, address(merkleVCA).code.length, "MerkleVCA contract not created");
```

Comment:
Any reason why you changed this from `assertGt` to `assertLt`?

Reply:
It's clearer to see the comparison to zero when 0 is the first parameter

Anyway, it's a nitpick, feel free to undo it if you don't like it

Reply:
Lets keep it as it will grow on me. I have a preference towards comparing things as `a >= b` vs `a <= b`.

Reply:
In general, I have the same preference, but I feel like for zero or constant comparisons, this makes it neat.

But again, no strong preference either way